airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -14,22 +14,16 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-import sys
 import tempfile
 import warnings
 from typing import Any, Dict, Generator, List, Optional, Tuple, Union
 
+from kubernetes import client, config, watch
 from kubernetes.config import ConfigException
```

Comment:
Ignorable nit: `kubernetes.config` is kinda sorta being imported twice.

Reply:
Yeah you're right. The import cleanup was a little bit me, but mostly isort. I can't take all the credit 🙂

Reply:
Shhhh!  Don't tell anyone :D